services/QuillLMS/engines/evidence/lib/evidence/open_ai/chat.rb

```diff
@@ -20,7 +19,7 @@ class Chat < Evidence::ApplicationService
 
       attr_reader :system_prompt, :entry, :history, :temperature, :model
 
-      def initialize(system_prompt:, entry:, history: [], temperature: 0.5, model: DEFAULT_MODEL)
+      def initialize(system_prompt:, entry:, history: [], temperature: 1, model: DEFAULT_MODEL)
```

Comment:
Is it worth setting up a `DEFAULT_TEMPERATURE` constant to match the pattern we're using for `DEFAULT_MODEL`?

Reply:
Will do!